packages/guui/dropdown/index.js

```diff
@@ -20,131 +20,139 @@ type Props = {
-        border: 'none',
-        lineHeight: 1.2,
-
-        fontSize: 14,
-        fontFamily: textSans,
-        color: palette.neutral['1'],
-        transition: 'color 80ms ease-out',
-        padding: '6px 10px',
-        margin: '1px 0 0',
-        textDecoration: 'none',
-        ':hover': {
-            textDecoration: 'underline',
-        },
-        ':focus': {
-            textDecoration: 'underline',
-        },
-
-        ':after': {
-            content: '""',
-            display: 'inline-block',
-            width: '4px',
-            height: '4px',
-            transform: 'translateY(-2px) rotate(45deg)',
-            border: '1px solid currentColor',
-            borderLeft: 'transparent',
-            borderTop: 'transparent',
-            marginLeft: '6px',
-            verticalAlign: 'middle',
-            transition: 'transform 250ms ease-out',
-        },
-    },
-
-    input: {
-        ...screenReaderOnly,
-        ':checked + ul': {
-            display: 'block',
-        },
-    },
-
-    'button.expanded:after, label.expanded:after': {
-        transform: 'translateY(1px) rotate(-135deg)',
-    },
-
-    ul: {
-        zIndex: 1072,
-        listStyle: 'none',
-        backgroundColor: 'white',
-        padding: '6px 0',
-        boxShadow: '0 0 0 1px rgba(0,0,0,0.1)',
-        borderRadius: '3px',
-        position: 'absolute',
-        right: '0',
-        width: '200px',
-        display: 'none',
-    },
-
-    'ul.expanded': {
-        display: 'block',
-    },
-
-    'li a': {
-        display: 'block',
-        padding: '10px 18px 15px 30px',
-    },
-
-    a: {
-        fontSize: '15px',
-        fontFamily: textSans,
-        color: palette.neutral['1'],
-        lineHeight: 1.2,
-        position: 'relative',
-        transition: 'color 80ms ease-out',
-        padding: '6px 10px',
-        margin: '-1px 0 0 0',
-        textDecoration: 'none',
-        ':hover': {
-            textDecoration: 'underline',
-        },
-        ':focus': {
-            textDecoration: 'underline',
-        },
-
-        ':before': {
-            content: '""',
-            borderTop: '1px solid #ededed',
-            display: 'block',
-            position: 'absolute',
-            top: 0,
-            left: '30px',
-            right: 0,
-        },
-    },
-
-    // no top border for first item
-    'li:first-child a:before': {
-        content: 'none',
-    },
-
-    'a:hover': {
-        backgroundColor: '#ededed',
-        textDecoration: 'none',
-    },
-
-    'a.active': {
-        fontWeight: 'bold',
-
-        ':after': {
-            content: '""',
-            border: '2px solid #c70000',
-            borderTop: 0,
-            borderRight: 0,
-            position: 'absolute',
-            top: '14px',
-            left: '10px',
-            width: '10px',
-            height: '4px',
-            transform: 'rotate(-45deg)',
-        },
-    },
-});
+const input = css`
+    // TODO re-add screen-reader only mixin
```

Comment:
At the moment the screenReaderOnly export is defined as a simple object. We should probably rewrite this as a css export but I wasn't sure if to do that here or separate PR.

Reply:
Ah yeah interesting, I guess the export and all the places it's used would need to be modified in a single pull request, unless we export an Object syntax version of `screenReaderOnly` and a TTL version temporarily and remove the object once all dependants are up to date.